### PR TITLE
completed ticket 9

### PR DESCRIPTION
**Pull request by t-cbailey**
9. DELETE /api/comments/:comment_id

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -338,4 +338,32 @@ describe.only("App", () => {
       });
     });
   });
+  describe("/api/comments", () => {
+    describe("/api/comments/:comment_id", () => {
+      test("DELETE 204 - should delete a comment with given comment_id and return no content", () => {
+        return request(app)
+          .delete("/api/comments/1")
+          .expect(201)
```
Comment (cjplatten) [inline __tests__/app.test.js]: The test name says 204 but you are expecting 201 here

File: __tests__/app.test.js [hunk 2]
```diff
@@ -338,4 +338,32 @@ describe.only("App", () => {
...
+          .then((res) => {
+            expect(res.body.length).toBe(0);
+          });
+      });
+      test("GET 404 when passed anything other than a number", () => {
+        return request(app)
+          .get("/api/comments/nonsense")
+          .expect(404)
```
Comment (cjplatten) [inline __tests__/app.test.js]: This should be 400

File: controllers.js [hunk 3]
```diff
@@ -71,4 +66,11 @@ exports.patchReviewVotesByRevID = (req, res, next) => {
     })
     .catch(next);
 };
-
+exports.deleteCommentById = (req, res, next) => {
+  const id = req.params.comment_id;
+  removeComment(id)
+    .then((output) => {
+      res.status(201).send(output);
```
Comment (cjplatten) [inline controllers.js]: there should not be a response body sent back

**Review by cjplatten — CHANGES_REQUESTED** [review]
See comments

**Comment by two-triangles** [comment]
Looks good, happy to merge!

**Pull request merged**